### DISCUSSION TITLE
Frontend Design

### DISCUSSION THREAD

**Discussion by devsoftzz** [WebMetadataRetrieval/15-Web_Metadata_Retrieval#29] [Ideas]
Designing the Home page similar to Github Home Page will look good. What do you think?

![Screenshot (121)](https://user-images.githubusercontent.com/50715966/114499968-c9115e80-9c44-11eb-9d5f-ab312a323beb.png)

**Comment by mananjoshi2000** [comment]
Yes, a minimalist design would be great.

**Comment by harshilg10** [comment]
Here is a design to start the implementation.. 
https://www.figma.com/file/XCsPfxMOsmiEsSwWxhuNH4/Untitled?node-id=0%3A1
Give your views and feedback for any changes.

**Reply by devsoftzz** [reply]
Looks Nice 🙌. Good Job
@ahirthecoder @Keyurkheni09 @shrutiagrawal09 @mananjoshi2000 start the implementation. Try to finish it before 20th April

**Reply by mananjoshi2000** [reply]
Looks cool! 😎 Will start the implementation and commit some of the pages by today

**Reply by shivam1shroff** [reply]
Is it possible that our page is also in dark theme as shown in the above image of GitHub?

**Reply by mananjoshi2000** [reply]
We will first focus on MVP, then we can do all the add ons

**Reply by Isha-Desai** [reply]
Kudos!! this design looks good.
